cluster/cluster.go

```diff
@@ -112,6 +118,7 @@ func Join(
 	probeInterval time.Duration,
 	reconnectInterval time.Duration,
 	reconnectTimeout time.Duration,
+	refreshInterval time.Duration,
```

Comment:
As this is not configurable via a command line flag anymore, there is no reason for the parameter, right?